cmd/manager/main.go

```diff
@@ -52,13 +52,6 @@ func printVersion() {
 
 func main() {
 	flag.Parse()
-
-	// The logger instantiated here can be changed to any logger
-	// implementing the logr.Logger interface. This logger will
-	// be propagated through the whole operator, generating
-	// uniform and structured logs.
-	logf.SetLogger(logf.ZapLogger(false))
-
```

Comment:
why are we removing this? will it cause any logging format change in operator logs?

Reply:
good catch - missed a local commit here